engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -407,16 +412,19 @@ object EngineJobExecutionActor {
   private[execution] case object NoData extends EJEAData
 
   private[execution] case class ResponsePendingData(jobDescriptor: BackendJobDescriptor,
-                                                    bjeaProps: Props,
-                                                    hashes: Option[Try[CallCacheHashes]] = None) extends EJEAData
+                                               bjeaProps: Props,
+                                               hashes: Option[Try[CallCacheHashes]] = None,
+                                               wdlValueSimpletons: Option[Seq[WdlValueSimpleton]] = None) extends EJEAData
```

Comment:
As far as I can tell, this is never actually read? It looks like it gets passed immediately from the `Fetch` actor to the `BackendCopy` actor, so we don't need to preserve it across states?